clients/js/src/types.ts

```diff
@@ -13,4 +13,36 @@ export type DimensionConfig = {
 
 export type IIsObject = {
     (item: any): boolean;
-}
\ No newline at end of file
+}
+
+export enum VariantType {
+    CONTROL = "CONTROL",
+    EXPERIMENTAL = "EXPERIMENTAL",
+}
+
+export type Variant = {
+    id: String,
+    overrides: Object,
+    variant_type: VariantType,
+}
+
+export type Variants = Array<Variant>;
+
+export enum ExperimentStatusType {
+    CREATED = "CREATED",
+    INPROGRESS = "INPROGRESS",
+    CONCLUDED = "CONCLUDED",
+}
+
+export type Experiment = {
+    variants: Variants,
+    name: String,
+    id: String,
+    traffic_percentage: number,
+    context: Object,
```

Comment:
Can context be a concrete type as well? It does have a definition in rust that can be used here.

Reply:
Its of raw jsonlogic context type